Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Models/Nations.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Know_Your_Nation_Speedy.Models
+{
+    public class Nations
+    {
+        public Nations()
+        {
+            Initialize();
+        }
+        public int NationsId { get; set; }
+        public string Name { get; set; }
+        public string ShortDescription { get; set; }
+        public HashSet<Articles> Article { get; set; }
+        public void Initialize()
+        {
+            Article = new HashSet<Articles>();
```

Comment:
if you're only initializing within your class, consider making "Article" an internal set. If you're planning to initialize from outside of your class you can leave as is

Reply:
so what you're saying is declare it without the initialize method? 
eg Icoll Articles = new HashSet();
wont it be better this way, so to save memory for only when the class is used?

Reply:
what you can do is just initialize it in the constructor instead of calling a method, but you can switch back to the method if you want once you start overloading the constructor

Reply:
okay will keep this in mind